app/src/marketplace/containers/ProductPage/PurchaseModal.jsx

```diff
@@ -109,7 +109,7 @@ export const PurchaseDialog = ({ productId, api }: Props) => {
             let succeeded = false
 
             try {
-                await createIdentity(account || 'Account name')
+                await connectIdentity(account || 'Account name')
```

Comment:
this created a new account instead of connecting the current one. oops 😬

Reply:
👏